### DISCUSSION TITLE
PyMuPDF not catching character `𠮟`

### DISCUSSION THREAD

**Discussion by vinhtq115** [pymupdf/PyMuPDF#1794] [Q&A]
## Describe the bug (mandatory)
For some reason, PyMuPDF won't catch `𠮟` character.

## To Reproduce (mandatory)
Example files:
[Archive.zip](https://github.com/pymupdf/PyMuPDF/files/9027673/Archive.zip)

```
import fitz

# doc = fitz.Document('aozorabunko_43768.pdf')
# page = doc[127]

doc = fitz.Document('aozorabunko_04508.pdf')
page = doc[197]

results = page.get_text('text')

for c in results:
    if c == '𠮟':
        print('Found')
```
When running this script, it doesn't output anything (since PyMuPDF doesn't catch that character). This character is recognized when I open the file in Firefox.

## Your configuration (mandatory)
 - Ubuntu 20.04.4 LTS
 - Python 3.8.13
 - PyMuPDF version 1.20.1 (wheel)

**Comment by JorjMcKie** [comment]
Adobe Acrobat, XPDF-Reader and others don't find it either.
So it has nothing to do with (Py-) MuPDF and is no bug.

Note:
The fact that you can **_see_** that character in a viewer has **_nothing_** to do with whether you can extract it.

**Comment by JorjMcKie** [comment]
FYI PyMuPDF extracts that ominous character as `chr(2975) = "ட"`.
Interestingly,`hex(2975) = 0xb9f`, whereas the complete character code is `0x20b9f`.
So if you search for **_this_**, you will find the right places.
Here is more info about this character. It seems to play a very special, unique role in Japanese scripts:

> ... This resulted in one joyo-kanji being in Extension B block if you strictly need to reproduce the appearance listed in the new official Joyo-kanji list. People usually use the original JIS X 0208 version without even noticing it. This joyo-kanji is 𠮟: it is U+20B9F (in Supplementary Ideographic Plane; SIP), but many people use 叱 U+53F1 which has been available since the plain old JIS X 0208. See [this article](https://hydrocul.github.io/wiki/blog/2014/1201-shikaru.html) (in Japanese).
